allensdk/brain_observatory/behavior/data_objects/metadata/behavior_metadata/project_id.py

```diff
@@ -0,0 +1,31 @@
+from pynwb import NWBFile
+
+from allensdk.core import DataObject
+from allensdk.core import \
+    LimsReadableInterface
+from allensdk.internal.api import PostgresQueryMixin
+from allensdk.core import NwbReadableInterface
+
+
+class ProjectId(DataObject, LimsReadableInterface, NwbReadableInterface):
+    def __init__(self, project_id: int):
+        super().__init__(name='project_id', value=project_id)
+
+    @classmethod
+    def from_lims(cls, behavior_session_id: int,
+                  lims_db: PostgresQueryMixin) -> "ProjectId":
+        query = f"""
+            SELECT bs.project_id AS project_id
+            FROM behavior_sessions bs
+            WHERE bs.id = {behavior_session_id}
+        """
+        project_id = lims_db.fetchone(query, strict=False)
```

Comment:
The type of the value seems inconsistent. Here, the code assumes it can be null. While in `from_nwb` it fills with -1. I think it should be consistently None or -1.

Reply:
Yeah, this was a set of spot fixes that that were used to get the unittests/already released data to behave consistently. I'll take a loop through the code and fixup the return types and make them consistent for `-1`.